apps/server-asset-sg/src/features/users/user.repo.ts

```diff
@@ -66,7 +66,7 @@ export class UserRepo implements Repo<User, UserId, UserData & { oidcId: string
           isAdmin: data.isAdmin,
           workgroups: {
             deleteMany: {
-              workgroupId: { notIn: data.workgroups?.map((workgroup) => workgroup.workgroupId) },
+              workgroupId: {},
```

Comment:
We ned to delete all UserWorkgroups for this user, not just the ones that have been removed, because their roles can also be updated, and this changes was not accounted for with the previous code